committeeoversightapp/models.py

```diff
@@ -498,13 +512,16 @@ def title_field(self):
 
     def save(self, *args, **kwargs):
         title = str(getattr(self, self.title_field))
+
+        if not title:
```

Comment:
I feel like there's probably a more elegant way to handle there being two possible sources for the site `title`... open to ideas @hancush!

Reply:
You could do 
```python
str(getattr(self, self.title_field) or getattr(self, self.fallback_title_field))
```

but I wouldn't do anything fancier than that.